tests/testlib/wakunode.nim

```diff
@@ -38,7 +38,7 @@ proc defaultTestWakuNodeConf*(): WakuNodeConf =
     maxConnections: 50,
     maxMessageSize: "1024 KiB",
     clusterId: 0,
-    pubsubTopics: @["/waku/2/rs/0/0"],
+    shards: @[uint16(0)],
```

Comment:
Maybe :)?
```suggestion
    shards: @[DefaultShardId],
```

Reply:
Ooh great one! Will also do the same for the clusterId :)